.pyup.yml

```diff
@@ -0,0 +1,4 @@
+schedule: "every week"
+requirements:
+  - Pipfile
```

Comment:
This repo uses a `Pipfile.lock` so we would either need to have pyup monitor the lock file, or drop the lock file.

Reply:
Fixed, I hope!